### PR TITLE
Bug/arth

### PR DESCRIPTION
**Pull request by probablyArth**

### CODE REVIEW

**Comment by probablyArth** [comment]
solves #9

File: output.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (MaximilianSoerenPollak) [inline output.txt]: Not quite sure if this was an intentional upload, if so what is it for?
If it was accidental, please remove.

File: models/tui.go [hunk 2]
```diff
@@ -33,30 +35,176 @@ func (m taskGroupsModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
 	case tea.KeyMsg:
 		switch keyPress := msg.String(); keyPress {
 		case "ctrl+c":
 			return m, tea.Quit
 		}
 		switch m.state {
 		case 0:
-			m, cmd, check := taskGroupViewState0(m, msg)
-			if !check {
-				return m, cmd
```
Comment (MaximilianSoerenPollak) [inline models/tui.go]: What was the rational behind reversing the split up again? Was it too convoluted?
Can you explain the thought process?

**Review by MaximilianSoerenPollak — COMMENTED** [review]
Just left some comments with questions. But other than those looks good.

**Review by MaximilianSoerenPollak — CHANGES_REQUESTED** [review]
Nice, if you could just remove the unwatned .txt file as well then this is GTM.

**Review by MaximilianSoerenPollak — APPROVED** [review]
Looks good.

LGTM :)

**Pull request merged**